src/CHANGES.txt

```diff
@@ -165,6 +165,9 @@ RELEASE 3.1.0.alpha.yyyymmdd - NEW DATE WILL BE INSERTED HERE
       Three uses of variables not defined are changed.
     - Some script changes in trying to find scons engine
     - Update (pep8) configure-cache script, add a --show option.
+    - Improve finding of Microsoft compiler: add a 'products' wildcard
```

Comment:
Add a note this is MS VC 2017 Build tools.  (it may or may not find any older versions..)